Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests.cs

```diff
@@ -44,7 +44,7 @@ public BasicVisualVerificationTests(ITestOutputHelper testOutputHelper)
 
                 // Here we check that the rendered content visually equals the reference image within a given error
                 // percentage. You can read more about this in the AssertVisualVerificationApproved method documentation.
-                context.AssertVisualVerificationApproved(navbarElementSelector, 8, cropRegion);
```

Comment:
Yes, you are right, it passes as expected.

Note, that the pixelErrorPercentageThreshold is 6.

 - With `SiteName = "Lombiq'sasfdasdf asdf"` and OS=Windows 10, the stats:
```
calculated differences:
    absoluteError=677626,
    meanError=21.044285714285714,
    pixelErrorCount=1799,
    pixelErrorPercentage=5.58695652173913
```
Diff image:
![1_BasicVisualVerificationTests_VerifyNavbar_By_ClassName Contains _-navbar-brand-Diff](https://user-images.githubusercontent.com/15349881/175098122-40b143be-9068-4918-9575-5d1ed6a05295.png)
 - With `SiteName = "'sasfdasdf asdfLombiq's OSOCE - UI Testing"` and OS=Windows 10, the stats:
```
calculated differences:
    absoluteError=1443192,
    meanError=44.81962732919255,
    pixelErrorCount=3738,
    pixelErrorPercentage=11.608695652173912
```
Diff image:
![2_BasicVisualVerificationTests_VerifyNavbar_By_ClassName Contains _-navbar-brand-Diff](https://user-images.githubusercontent.com/15349881/175098427-6d5c04d4-586b-406b-a79a-4631cc934ea1.png)
 - With `SiteName = "Lombiq's OSOCE - UI Testing"` and OS=GH ubuntu-latest, the stats:
```
calculated differences:
absoluteError=298597,
meanError=9.273198757763975,
pixelErrorCount=1705,
pixelErrorPercentage=5.295031055900621
```
Diff image:
![image](https://user-images.githubusercontent.com/15349881/175098831-f7065ea0-9bd8-4ce4-8267-b7e0df3a1c1a.png)

I can't set a lower threshold, because of the different font between Windows and Linux.
There is a lower threshold in `VerifyHomePage` because there is a lower text/area ratio.

Reply:
Let's have a value that will fail the test if something like more than a word is changed. 5.3 would be quite peculiar and thus leave people wondering... Though at this point, it occurred to me that this is actually a good idea, because we can use this as a teachable example of what needs to be considered when fine-tuning these values. So let's use 5.3, just please add a comment explaining why this exact value.